App.tsx

```
@@ -279,9 +313,34 @@ const ConfigurationPage = ({ repos, setRepos, user }: { repos: Repository[], set
                     </button>
                     <h1 className="text-lg font-bold">{activeRepoId ? 'Edit Repository' : 'Add Repository'}</h1>
                 </div>
-                <button onClick={saveRepo} className="text-primary font-bold text-sm">Save</button>
+                <div className="flex items-center gap-2">
+                    {activeRepoId && (
+                        <button
+                            onClick={deleteRepo}
+                            disabled={saving}
+                            className="text-red-500 font-bold text-sm disabled:opacity-50 disabled:pointer-events-none flex items-center gap-1 border border-red-500/40 rounded-md px-2 py-1"
+                        >
+                            <span className="material-symbols-outlined text-base">delete</span>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)